packages/suite/src/components/suite/Labeling/components/AddressLabeling.tsx

```diff
@@ -1,24 +1,22 @@
 import React from 'react';
 import * as accountUtils from '@wallet-utils/accountUtils';
 import { useSelector } from '@suite-hooks';
-import AccountLabel from '../Account';
+import { AccountLabeling } from './AccountLabeling';
 
 // DeviceLabel? / WalletLabel? / Account #N / AccountType?
```

Comment:
not sure what this comments is probably some suggestion for better name? I think we can delete it

Reply:
done